expan/core/statistics.py

```diff
@@ -134,6 +134,37 @@ def sample_size(x):
     return len(x) - x_nan
 
 
+def estimate_sample_size(x, mde, n, r, alpha=0.05, beta=0.2):
```

Comment:
It's an isolated method for now and we don't expose this to result yet, right?

Reply:
yes, it has to be called separately